stylix/autoload.nix

```
@@ -8,7 +8,45 @@ builtins.concatLists (
     path: kind:
     let
       file = "${inputs.self}/modules/${path}/${for}.nix";
+      value = import file;
```

```suggestion
      fn = import file;
```

Technically, we don't know it's a function yet when assigning this binding, however in all the places we _use_ `value`, we _have_ checked it is a function.

So maybe this is a better name?

Could also do `maybeFn`, but that could get wordy.

Or `module`, but that could conflict with L41.

Although, L41 could be inlined into the `imports` list: 
```nix
imports = [
  (value (args // extraArgs // { inherit mkTarget; }))
];
```

_Personally_ I think something like `fn` or `moduleFn` is best, but as with all naming it's very subjective!

---

> Technically, we don't know it's a function yet when assigning this binding, however in all the places we _use_ `value`, we _have_ checked it is a function.
>
> So maybe this is a better name?
>
> Could also do `maybeFn`, but that could get wordy.
>
> Or `module`, but that could conflict with L41.
>
> Although, L41 could be inlined into the `imports` list:
>
> ```nix
> imports = [
>   (value (args // extraArgs // { inherit mkTarget; }))
> ];
> ```

Although I am biased because I generally [prefer inlining code](http://number-none.com/blow/blog/programming/2014/09/26/carmack-on-inlined-code.html), I am fine with the coding style of self documenting the code with intermediate variables.

Since the `module` identifier looks very appropriate, I prefer inlining the current `module` variable and using `module` as:

```suggestion
      module = import file;
```